### PR TITLE
Uploading Scripts to ADSync Branch

### PR DESCRIPTION
**Pull request by chboeh**
Fixes #

## Azure Active Directory Log Analytics Enrichment (Script)
This scripts is a manual ingestion of Azure Active Directory data (unfiltered) to Log Analytics.
Requirements are setting up and configuring service account that has access to Azure Active Directory

## Azure Automation AAD Import (Script)
This script is a automation ingestion of AAD Data (unfiltered) into Log Analytics.
Requirements are allowing the Automation account access to the Azure AD (GraphAPI) Read Directory permissions
Note: Due to some data set size(memory) issue it was configured to ingest 10 users at a time, you can expand this depending on what information you're wanting to pull from AAD

## Azure Active Directory Function (Copy of script example)
This script is using API's from Azure AD with Graph Security API and ingesting data from AAD into Log Analytics much like the Azure Automation script without modules.
  -
  -
  -

### CODE REVIEW

File: Solutions/AzureActiveDirectoryUserDetails/AzureAutomationAADimport.ps1 [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        $ErrorMessage = "Connection $connectionName not found."
+        throw $ErrorMessage
+    } else{
+        Write-Error -Message $_.Exception
+        throw $_.Exception
+    }
+} 
+
+$CustomerID = 'WORKSPACE ID'
+$SharedKey = 'WORKSPACE PRIMARY KEY'
```
Comment (juliango2100) [inline Solutions/AzureActiveDirectoryUserDetails/AzureAutomationAADimport.ps1:65]: This should use encrypted variables via azure automation
https://docs.microsoft.com/en-us/azure/automation/shared-resources/variables

Comment (juliango2100) [inline Solutions/AzureActiveDirectoryUserDetails/AzureAutomationAADimport.ps1:65]: This should use encrypted variables via azure automation
https://docs.microsoft.com/en-us/azure/automation/shared-resources/variables

Comment (preetikr) [reply Solutions/AzureActiveDirectoryUserDetails/AzureAutomationAADimport.ps1:65]: Hi @chboeh - Solutions folder has a specific purpose, hence let's avoid creating these in Solutions folder. You can get these into a new sub-folder under tools or if these are for data ingestion, you can also add these to the DataConnectors folder of the GitHub repo. Let's sync on these changes and close on it as these are pretty old. Thanks.

**Comment by preetikr** [comment]
Hi @chboeh - please have the folder update for this to be under data connectors - thanks.

**Comment by preetikr** [comment]
@NoamLandress - is it possible to work with Alaa to review and approve this? Thanks.

**Pull request closed without merging**